swagger.yaml

```diff
@@ -0,0 +1,111 @@
+swagger: "2.0"
+info:
+  description: "An API used to query information about resources"
+  version: "1.0.0"
+  title: "ONS Resource API"
+  license:
+    name: "Open Government Licence v3.0"
+    url: "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
+basePath: "/v1"
+tags:
+- name: "Resources"
```

Comment:
Resources is very a general and overloaded term. It may be worth having a discussion as a group to determine if there is something more specific we can call it.